docs/hooks_example.rst

```diff
@@ -0,0 +1,43 @@
+EOX_HOOKS Settings Example
+===========================
+
+.. code-block:: yaml
+
+    USE_EOX_HOOKS: True
+    EOX_HOOKS_DEFINITIONS:
+    {
+        "post_register":
+        {
+            "action": "post_to_webhook_url",
+            "config":
+            {
+                "fields": { "recipient_email": "user.pii.email" },
+                "send_certificate_data": False,
+                "url": "https://webhook.site",
+            },
+            "fail_silently": False,
+            "module": "eox_hooks.actions",
+        },
+        "post_session_login":
+        {
+            "action": "post_to_webhook_url",
+            "config":
+            {
+                "fields": { "recipient_email": "user.pii.email" },
+                "url": "https://webhook.site/",
+            },
+            "fail_silently": True,
+            "module": "eox_hooks.actions",
+        },
+        "post_enrollment":
+        {
+            "action": "post_to_webhook_url",
```

Comment:
Please could we add examples of other actions?

Reply:
It'd be interesting to add a similar file as in trigger_event_examples as Bryann suggests.

Reply:
Thanks, @mariajgrimaldi and @BryanttV. I added the information based on the old readme, but please feel free to reopen the conversation if necessary.

Btw, we need to check in another moment about `trigger_grades_assignment` and `trigger_enrollments_creation` because we are not testing them and don't have updated information on whether those actions work correctly.